cli/delete_test.go

```diff
@@ -209,4 +218,225 @@ func TestDelete(t *testing.T) {
 		cancel()
 		<-doneChan
 	})
+
+	t.Run("Workspace delete permissions", func(t *testing.T) {
```

Comment:
suggest mentioning this is related to prebuilds in the test name

Reply:
Addressed in https://github.com/coder/coder/pull/18333/commits/f811778881c619807d394fa830f4d22c16264ee2